### ISSUE TITLE
Focus outlines not shown in popped-out editor windows

### ISSUE THREAD

**Issue by gtritchie** [rstudio/rstudio#7882]
### System details

    RStudio Edition : Desktop or Server
    RStudio Version : 1.4.869
    OS Version      : any
    R Version       : any

### Steps to reproduce the problem
- Start the IDE, create or load an R script
- Move that to an external editor window ("Show document in new window")
- Click on a blank area of the toolbar as pointed out here (or turn on on the accessibility option "Tab key always moves focus" and use Shift+Tab to move focus to toolbar):

<img width="688" alt="screenshot of external editor window with arrow pointing at blank area in toolbar" src="https://user-images.githubusercontent.com/10569626/94225556-23fe8680-feaa-11ea-8418-3a4dc4a6a471.png">

- Use the Tab key to move focus around the toolbar

### Describe the problem in detail
- No focus indicators on the controls (except the checkbox)

### Describe the behavior you expected
- Focus indicators like are shown in 1.3

- [x] I have read the guide for [submitting good bug reports](https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports).
- [x] I have installed the latest version of RStudio, and confirmed that the issue still persists.
- [x] If I am reporting a RStudio crash, I have included a [diagnostics report](https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report).
- [x] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

**Comment by dfalty** [comment]
Verified fixed on v1.4.874. Tab key will cause focus indicators to go around the controls.
